anyway/localization.py

```diff
@@ -222,3 +222,179 @@ def get_city_name(symbol_id, lang: str = "he") -> Optional[str]:
         return _cities.loc[symbol_id, column_to_fetch]
     except Exception:
         return None
+
+
+def join_hebrew_strings(strings, sep_a=' ,', sep_b=' ו-'):
+    if len(strings) < 2:
+        return ''.join(strings)
+    elif len(strings) == 2:
+        return sep_b.join(strings)
+    else:
+        return sep_a.join(strings[:-1]) + sep_b + strings[-1]
+
+def to_hebrew(output):
+    def get_injured_count_hebrew(data):
+        severity_light_count = data.get('light_injured_count')
+        if severity_light_count == 0:
+            severity_light_count_text = ''
+        elif severity_light_count == 1:
+            severity_light_count_text = 'פצוע אחד קל'
+        else:
+            severity_light_count_text = f'{severity_light_count} פצועים קל'
+
+        severity_severe_count = data.get('severe_injured_count')
+        if severity_severe_count == 0:
+            severity_severe_count_text = ''
+        elif severity_severe_count == 1:
+            severity_severe_count_text = 'פצוע אחד קשה'
+        else:
+            severity_severe_count_text = f'{severity_severe_count} פצועים קשה'
+
+        severity_fatal_count = data.get('killed_count')
+        if severity_fatal_count == 0:
+            severity_fatal_count_text = ''
+        elif severity_fatal_count == 1:
+            severity_fatal_count_text = 'הרוג אחד'
+        else:
+            severity_fatal_count_text = f'{severity_fatal_count} הרוגים'
+        return join_hebrew_strings(list(filter(lambda s: s != '', [severity_fatal_count_text, severity_severe_count_text, severity_light_count_text])))
+
+
+    def most_severe_accidents_table_hebrew():
+        date_range = output.get('meta').get('dates_comment').get('date_range')
+        most_severe_accidents_table = list(filter(lambda widget: widget.get('name') == 'most_severe_accidents_table', output.get('widgets')))[0]
+        most_severe_accidents_table_items = most_severe_accidents_table.get('data').get('items')
+
+        if len(most_severe_accidents_table_items) == 1:
+            text = " התאונה החמורה האחרונה שהתרחשה בכביש"
+        elif len(most_severe_accidents_table_items) > 1:
+            text = f"\u202b{len(most_severe_accidents_table_items)}\u202c התאונות החמורות האחרונות שהתרחשו בכביש "
```

Comment:
If `most_severe_accidents_table_items` cannot be empty, the if part in line 270 is redundant. Otherwise, the case of empty value needs to be handled